packages/application/src/index.ts

```diff
@@ -613,7 +714,20 @@ namespace Private {
     /**
      * The function which activates the plugin.
      */
-    readonly activate: (app: any, ...args: any[]) => any;
+    readonly activate: (
+      app: Application<Widget>,
+      ...args: Token<any>[]
+    ) => Token<any> | Promise<Token<any>>;
```

Comment:
The `activate` function returns `any` item that the `Token` represents, not the `Token` itself.